bedrock/settings/base.py

```diff
@@ -1430,11 +1429,13 @@ def get_default_gateway_linux():
 
 DEAD_MANS_SNITCH_URL = config('DEAD_MANS_SNITCH_URL', default='')
 
-RAVEN_CONFIG = {
-    'dsn': config('SENTRY_DSN', default=''),
-    'site': '.'.join(x for x in [APP_NAME, CLUSTER_NAME] if x),
-    'release': config('GIT_SHA', default=''),
-}
+SENTRY_DSN = config("SENTRY_DSN", default='')
```

Comment:
nit: `s/"/'/` (also below)

and when we convert to using Black we can changed everything back :rofl: